src/libs/OptionsListUtils.js

```diff
@@ -372,6 +373,40 @@ function getAllReportErrors(report, reportActions) {
     return allReportErrors;
 }
 
+/**
+ * Get the preview message to be displayed in the option list.
+ * @param {Object} report
+ * @param {Object} reportAction
+ * @param {Boolean} isPreviewMessageForParentChatReport
```

Comment:
Looks like it is an optional argument.

```suggestion
 * @param {Boolean} [isPreviewMessageForParentChatReport]
```